src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/DcirMedicalActExtractor.scala

```diff
@@ -62,3 +62,16 @@ object DcirMedicalActExtractor extends DcirExtractor[MedicalAct] {
   override def extractWeight(r: Row): Double = 1.0
 
 }
+
+
+object DcirMedicalActExtractor extends DcirActExtractor  {
+  override val columnName: String = ColNames.CamCode
+  override val eventBuilder: EventBuilder = DcirAct
+}
+
+
+object DcirBiologyActExtractor extends DcirActExtractor  {
+  override val columnName: String = ColNames.BioCode
+  override val eventBuilder: EventBuilder = BiologyDcirAct
+  override def code = (row: Row) => row.getAs[Double](columnName).toString
```

Comment:
Why is this here? Isn't already implemented?

Reply:
code asks for a String but this column is a Double